snagsby/sources.py

```diff
@@ -99,6 +133,6 @@ def _parse_sources_str(sources_str):
 
 def parse_sources(sources_str):
     return [
-        S3Source(source)
+        S3Source(source) if source.startswith('s3://') else SMSource(source)
```

Comment:
We might want to be more explicit about only accepting inputs that start with either s3:// or sm:// ?